libcudacxx/benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+include(${CMAKE_SOURCE_DIR}/benchmarks/cmake/CCCLBenchmarkRegistry.cmake)
+
+cccl_get_nvbench_helper()
```

Comment:
This adds a lite dependency on `nvbench_helper` for `libcudacxx` in `ci/inspect_changes.sh` or `ci/project_files_and_dependencies.yaml` (depending on when #6535 lands)